Add some diagnostics

mikehaney24 opened the pull request:
Hello! I'm one of the authors of Firebase Crash Reporting. I'm trying to figure out why you're having trouble uploading crashes. Can you accept this change, re-run, and then update https://github.com/UglyBlueCat/quickstart-ios/blob/master/crashreporting/upload_sym_error with the output of that?

I was prohibited from updating to High Sierra for a while on my work computer and wasn't able to repro on Sierra. Now that I've finally been allowed to upgrade to High Sierra, I tried again and still couldn't repro.

I also just want to say that I appreciate your efforts to give as much information as possible!

Lastly, I'm also interested in the output of the following two commands:
which curl
curl --version

Thanks!

UglyBlueCat commented:
Hi Micheal,

I've saved the log into
https://github.com/UglyBlueCat/quickstart-ios/blob/master/crashreporting/Build%20CrashReportingExample_2018-01-09T11-00-59.rtf

I don't know why I didn't see that save button before, but now you have the
whole part of the log relevant to the issue.

here's the output of the two commands you requested:

$ which curl
/usr/bin/curl
$ curl --version
curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
zlib/1.2.11 nghttp2/1.24.0
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3
pop3s rtsp smb smbs smtp smtps telnet tftp
Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL
libz HTTP2 UnixSockets HTTPS-proxy

I should have told you that we have now installed crashlytics, as per your
colleagues suggestion, and it is working fine, so we have removed firebase
crash reporting.

We figured google probably wouldn't pursue this issue as a priority, since
they appear to be phasing out firebase crash reporting.

You may close this issue if you wish, but I will continue to help you
resolve it if that is your wish, since there may be others with the issue
who can't (or won't) switch to crashlytics.

Thanks,
Robin


Robin Spinks
07710 520048
uglybluecat@googlemail.com

On 9 January 2018 at 00:29, Michael Haney <notifications@github.com> wrote:

> Hello! I'm one of the authors of Firebase Crash Reporting. I'm trying to
> figure out why you're having trouble uploading crashes. Can you accept this
> change, re-run, and then update https://github.com/
> UglyBlueCat/quickstart-ios/blob/master/crashreporting/upload_sym_error
> with the output of that?
>
> I was prohibited from updating to High Sierra for a while on my work
> computer and wasn't able to repro on Sierra. Now that I've finally been
> allowed to upgrade to High Sierra, I tried again and still couldn't repro.
>
> I also just want to say that I appreciate your efforts to give as much
> information as possible!
>
> Lastly, I'm also interested in the output of the following two commands:
> which curl
> curl --version
>
> Thanks!
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/UglyBlueCat/quickstart-ios/pull/1
> Commit Summary
>
>    - Add some diagnostics
>
> File Changes
>
>    - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
>    <https://github.com/UglyBlueCat/quickstart-ios/pull/1/files#diff-0>
>    (2)
>
> Patch Links:
>
>    - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
>    - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia>
> .
>

mikehaney24 commented:
Thanks for uploading the new log.

You're right--Crashlytics is the way to go, but we definitely don't want to
there to be any issues for our existing customers of Firebase Crash, so I'm
trying to fix this.

To that end, I've been trying to reproduce this issue and haven't been able
to. I'll keep trying and ask for more information if I need any. Thanks a
lot for your help so far!

On Tue, Jan 9, 2018 at 3:46 AM Robin Spinks <notifications@github.com>
wrote:

> Hi Micheal,
>
> I've saved the log into
>
> https://github.com/UglyBlueCat/quickstart-ios/blob/master/crashreporting/Build%20CrashReportingExample_2018-01-09T11-00-59.rtf
>
> I don't know why I didn't see that save button before, but now you have the
> whole part of the log relevant to the issue.
>
> here's the output of the two commands you requested:
>
> $ which curl
> /usr/bin/curl
> $ curl --version
> curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
> zlib/1.2.11 nghttp2/1.24.0
> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3
> pop3s rtsp smb smbs smtp smtps telnet tftp
> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL
> libz HTTP2 UnixSockets HTTPS-proxy
>
> I should have told you that we have now installed crashlytics, as per your
> colleagues suggestion, and it is working fine, so we have removed firebase
> crash reporting.
>
> We figured google probably wouldn't pursue this issue as a priority, since
> they appear to be phasing out firebase crash reporting.
>
> You may close this issue if you wish, but I will continue to help you
> resolve it if that is your wish, since there may be others with the issue
> who can't (or won't) switch to crashlytics.
>
> Thanks,
> Robin
>
>
> Robin Spinks
> 07710 520048
> uglybluecat@googlemail.com
>
> On 9 January 2018 at 00:29, Michael Haney <notifications@github.com>
> wrote:
>
> > Hello! I'm one of the authors of Firebase Crash Reporting. I'm trying to
> > figure out why you're having trouble uploading crashes. Can you accept
> this
> > change, re-run, and then update https://github.com/
> > UglyBlueCat/quickstart-ios/blob/master/crashreporting/upload_sym_error
> > with the output of that?
> >
> > I was prohibited from updating to High Sierra for a while on my work
> > computer and wasn't able to repro on Sierra. Now that I've finally been
> > allowed to upgrade to High Sierra, I tried again and still couldn't
> repro.
> >
> > I also just want to say that I appreciate your efforts to give as much
> > information as possible!
> >
> > Lastly, I'm also interested in the output of the following two commands:
> > which curl
> > curl --version
> >
> > Thanks!
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/UglyBlueCat/quickstart-ios/pull/1
> > Commit Summary
> >
> > - Add some diagnostics
> >
> > File Changes
> >
> > - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1/files#diff-0>
> > (2)
> >
> > Patch Links:
> >
> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute the
> thread
> > <
> https://github.com/notifications/unsubscribe-auth/AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-356261857>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCFgjHUqu_1w4KSGNAJ2aNMTUg_-dCqks5tI1F7gaJpZM4RXJia>
> .
>

mikehaney24 commented:
Hi Robin,

Odd request: can you please build and run 3 times and upload (or email me)
the output of the 3rd run? I really appreciate your help, and I think I'm
getting to the bottom of the issue.

On Tue, Jan 16, 2018 at 7:29 PM Michael Haney <mikehaney24@gmail.com> wrote:

> Thanks for uploading the new log.
>
> You're right--Crashlytics is the way to go, but we definitely don't want
> to there to be any issues for our existing customers of Firebase Crash, so
> I'm trying to fix this.
>
> To that end, I've been trying to reproduce this issue and haven't been
> able to. I'll keep trying and ask for more information if I need any.
> Thanks a lot for your help so far!
>
> On Tue, Jan 9, 2018 at 3:46 AM Robin Spinks <notifications@github.com>
> wrote:
>
>> Hi Micheal,
>>
>> I've saved the log into
>>
>> https://github.com/UglyBlueCat/quickstart-ios/blob/master/crashreporting/Build%20CrashReportingExample_2018-01-09T11-00-59.rtf
>>
>> I don't know why I didn't see that save button before, but now you have
>> the
>> whole part of the log relevant to the issue.
>>
>> here's the output of the two commands you requested:
>>
>> $ which curl
>> /usr/bin/curl
>> $ curl --version
>> curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
>> zlib/1.2.11 nghttp2/1.24.0
>> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3
>> pop3s rtsp smb smbs smtp smtps telnet tftp
>> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB
>> SSL
>> libz HTTP2 UnixSockets HTTPS-proxy
>>
>> I should have told you that we have now installed crashlytics, as per your
>> colleagues suggestion, and it is working fine, so we have removed firebase
>> crash reporting.
>>
>> We figured google probably wouldn't pursue this issue as a priority, since
>> they appear to be phasing out firebase crash reporting.
>>
>> You may close this issue if you wish, but I will continue to help you
>> resolve it if that is your wish, since there may be others with the issue
>> who can't (or won't) switch to crashlytics.
>>
>> Thanks,
>> Robin
>>
>>
>> Robin Spinks
>> 07710 520048
>> uglybluecat@googlemail.com
>>
>> On 9 January 2018 at 00:29, Michael Haney <notifications@github.com>
>> wrote:
>>
>> > Hello! I'm one of the authors of Firebase Crash Reporting. I'm trying to
>> > figure out why you're having trouble uploading crashes. Can you accept
>> this
>> > change, re-run, and then update https://github.com/
>> > UglyBlueCat/quickstart-ios/blob/master/crashreporting/upload_sym_error
>> > with the output of that?
>> >
>> > I was prohibited from updating to High Sierra for a while on my work
>> > computer and wasn't able to repro on Sierra. Now that I've finally been
>> > allowed to upgrade to High Sierra, I tried again and still couldn't
>> repro.
>> >
>> > I also just want to say that I appreciate your efforts to give as much
>> > information as possible!
>> >
>> > Lastly, I'm also interested in the output of the following two commands:
>> > which curl
>> > curl --version
>> >
>> > Thanks!
>> > ------------------------------
>> > You can view, comment on, or merge this pull request online at:
>> >
>> > https://github.com/UglyBlueCat/quickstart-ios/pull/1
>> > Commit Summary
>> >
>> > - Add some diagnostics
>> >
>> > File Changes
>> >
>> > - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
>> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1/files#diff-0>
>> > (2)
>> >
>> > Patch Links:
>> >
>> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
>> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
>> >
>> > —
>> > You are receiving this because you are subscribed to this thread.
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute the
>> thread
>> > <
>> https://github.com/notifications/unsubscribe-auth/AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia
>> >
>> > .
>> >
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-356261857>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ABCFgjHUqu_1w4KSGNAJ2aNMTUg_-dCqks5tI1F7gaJpZM4RXJia>
>> .
>>
>

UglyBlueCat commented:
Hi Mike,

Do you still need this?

I'm sorry, but my work has become very busy as I approach a deadline.

Robin


Robin Spinks
07710 520048
uglybluecat@googlemail.com

On 18 January 2018 at 00:29, Michael Haney <notifications@github.com> wrote:

> Hi Robin,
>
> Odd request: can you please build and run 3 times and upload (or email me)
> the output of the 3rd run? I really appreciate your help, and I think I'm
> getting to the bottom of the issue.
>
> On Tue, Jan 16, 2018 at 7:29 PM Michael Haney <mikehaney24@gmail.com>
> wrote:
>
> > Thanks for uploading the new log.
> >
> > You're right--Crashlytics is the way to go, but we definitely don't want
> > to there to be any issues for our existing customers of Firebase Crash,
> so
> > I'm trying to fix this.
> >
> > To that end, I've been trying to reproduce this issue and haven't been
> > able to. I'll keep trying and ask for more information if I need any.
> > Thanks a lot for your help so far!
> >
> > On Tue, Jan 9, 2018 at 3:46 AM Robin Spinks <notifications@github.com>
> > wrote:
> >
> >> Hi Micheal,
> >>
> >> I've saved the log into
> >>
> >> https://github.com/UglyBlueCat/quickstart-ios/
> blob/master/crashreporting/Build%20CrashReportingExample_
> 2018-01-09T11-00-59.rtf
> >>
> >> I don't know why I didn't see that save button before, but now you have
> >> the
> >> whole part of the log relevant to the issue.
> >>
> >> here's the output of the two commands you requested:
> >>
> >> $ which curl
> >> /usr/bin/curl
> >> $ curl --version
> >> curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
> >> zlib/1.2.11 nghttp2/1.24.0
> >> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps
> pop3
> >> pop3s rtsp smb smbs smtp smtps telnet tftp
> >> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB
> >> SSL
> >> libz HTTP2 UnixSockets HTTPS-proxy
> >>
> >> I should have told you that we have now installed crashlytics, as per
> your
> >> colleagues suggestion, and it is working fine, so we have removed
> firebase
> >> crash reporting.
> >>
> >> We figured google probably wouldn't pursue this issue as a priority,
> since
> >> they appear to be phasing out firebase crash reporting.
> >>
> >> You may close this issue if you wish, but I will continue to help you
> >> resolve it if that is your wish, since there may be others with the
> issue
> >> who can't (or won't) switch to crashlytics.
> >>
> >> Thanks,
> >> Robin
> >>
> >>
> >> Robin Spinks
> >> 07710 520048
> >> uglybluecat@googlemail.com
> >>
> >> On 9 January 2018 at 00:29, Michael Haney <notifications@github.com>
> >> wrote:
> >>
> >> > Hello! I'm one of the authors of Firebase Crash Reporting. I'm trying
> to
> >> > figure out why you're having trouble uploading crashes. Can you accept
> >> this
> >> > change, re-run, and then update https://github.com/
> >> > UglyBlueCat/quickstart-ios/blob/master/crashreporting/
> upload_sym_error
> >> > with the output of that?
> >> >
> >> > I was prohibited from updating to High Sierra for a while on my work
> >> > computer and wasn't able to repro on Sierra. Now that I've finally
> been
> >> > allowed to upgrade to High Sierra, I tried again and still couldn't
> >> repro.
> >> >
> >> > I also just want to say that I appreciate your efforts to give as much
> >> > information as possible!
> >> >
> >> > Lastly, I'm also interested in the output of the following two
> commands:
> >> > which curl
> >> > curl --version
> >> >
> >> > Thanks!
> >> > ------------------------------
> >> > You can view, comment on, or merge this pull request online at:
> >> >
> >> > https://github.com/UglyBlueCat/quickstart-ios/pull/1
> >> > Commit Summary
> >> >
> >> > - Add some diagnostics
> >> >
> >> > File Changes
> >> >
> >> > - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
> >> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1/files#diff-0>
> >> > (2)
> >> >
> >> > Patch Links:
> >> >
> >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
> >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
> >> >
> >> > —
> >> > You are receiving this because you are subscribed to this thread.
> >> > Reply to this email directly, view it on GitHub
> >> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute the
> >> thread
> >> > <
> >> https://github.com/notifications/unsubscribe-auth/
> AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia
> >> >
> >> > .
> >> >
> >>
> >> —
> >> You are receiving this because you authored the thread.
> >> Reply to this email directly, view it on GitHub
> >> <https://github.com/UglyBlueCat/quickstart-ios/
> pull/1#issuecomment-356261857>,
> >> or mute the thread
> >> <https://github.com/notifications/unsubscribe-auth/ABCFgjHUqu_
> 1w4KSGNAJ2aNMTUg_-dCqks5tI1F7gaJpZM4RXJia>
> >> .
> >>
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-358496458>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4FNGTl3V9obVinkngattqZw22a9fqBks5tLpBjgaJpZM4RXJia>
> .
>

mikehaney24 commented:
If you're able to do it, we'd appreciate it and I'd try to use that
information to solve the problem. For now, after spending 5 full days
trying to reproduce this, I've moved on to other issues.

On Mon, Feb 12, 2018 at 2:57 AM Robin Spinks <notifications@github.com>
wrote:

> Hi Mike,
>
> Do you still need this?
>
> I'm sorry, but my work has become very busy as I approach a deadline.
>
> Robin
>
>
> Robin Spinks
> 07710 520048
> uglybluecat@googlemail.com
>
> On 18 January 2018 at 00:29, Michael Haney <notifications@github.com>
> wrote:
>
> > Hi Robin,
> >
> > Odd request: can you please build and run 3 times and upload (or email
> me)
> > the output of the 3rd run? I really appreciate your help, and I think I'm
> > getting to the bottom of the issue.
> >
> > On Tue, Jan 16, 2018 at 7:29 PM Michael Haney <mikehaney24@gmail.com>
> > wrote:
> >
> > > Thanks for uploading the new log.
> > >
> > > You're right--Crashlytics is the way to go, but we definitely don't
> want
> > > to there to be any issues for our existing customers of Firebase Crash,
> > so
> > > I'm trying to fix this.
> > >
> > > To that end, I've been trying to reproduce this issue and haven't been
> > > able to. I'll keep trying and ask for more information if I need any.
> > > Thanks a lot for your help so far!
> > >
> > > On Tue, Jan 9, 2018 at 3:46 AM Robin Spinks <notifications@github.com>
> > > wrote:
> > >
> > >> Hi Micheal,
> > >>
> > >> I've saved the log into
> > >>
> > >> https://github.com/UglyBlueCat/quickstart-ios/
> > blob/master/crashreporting/Build%20CrashReportingExample_
> > 2018-01-09T11-00-59.rtf
> > >>
> > >> I don't know why I didn't see that save button before, but now you
> have
> > >> the
> > >> whole part of the log relevant to the issue.
> > >>
> > >> here's the output of the two commands you requested:
> > >>
> > >> $ which curl
> > >> /usr/bin/curl
> > >> $ curl --version
> > >> curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
> > >> zlib/1.2.11 nghttp2/1.24.0
> > >> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps
> > pop3
> > >> pop3s rtsp smb smbs smtp smtps telnet tftp
> > >> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM
> NTLM_WB
> > >> SSL
> > >> libz HTTP2 UnixSockets HTTPS-proxy
> > >>
> > >> I should have told you that we have now installed crashlytics, as per
> > your
> > >> colleagues suggestion, and it is working fine, so we have removed
> > firebase
> > >> crash reporting.
> > >>
> > >> We figured google probably wouldn't pursue this issue as a priority,
> > since
> > >> they appear to be phasing out firebase crash reporting.
> > >>
> > >> You may close this issue if you wish, but I will continue to help you
> > >> resolve it if that is your wish, since there may be others with the
> > issue
> > >> who can't (or won't) switch to crashlytics.
> > >>
> > >> Thanks,
> > >> Robin
> > >>
> > >>
> > >> Robin Spinks
> > >> 07710 520048
> > >> uglybluecat@googlemail.com
> > >>
> > >> On 9 January 2018 at 00:29, Michael Haney <notifications@github.com>
> > >> wrote:
> > >>
> > >> > Hello! I'm one of the authors of Firebase Crash Reporting. I'm
> trying
> > to
> > >> > figure out why you're having trouble uploading crashes. Can you
> accept
> > >> this
> > >> > change, re-run, and then update https://github.com/
> > >> > UglyBlueCat/quickstart-ios/blob/master/crashreporting/
> > upload_sym_error
> > >> > with the output of that?
> > >> >
> > >> > I was prohibited from updating to High Sierra for a while on my work
> > >> > computer and wasn't able to repro on Sierra. Now that I've finally
> > been
> > >> > allowed to upgrade to High Sierra, I tried again and still couldn't
> > >> repro.
> > >> >
> > >> > I also just want to say that I appreciate your efforts to give as
> much
> > >> > information as possible!
> > >> >
> > >> > Lastly, I'm also interested in the output of the following two
> > commands:
> > >> > which curl
> > >> > curl --version
> > >> >
> > >> > Thanks!
> > >> > ------------------------------
> > >> > You can view, comment on, or merge this pull request online at:
> > >> >
> > >> > https://github.com/UglyBlueCat/quickstart-ios/pull/1
> > >> > Commit Summary
> > >> >
> > >> > - Add some diagnostics
> > >> >
> > >> > File Changes
> > >> >
> > >> > - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
> > >> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1/files#diff-0>
> > >> > (2)
> > >> >
> > >> > Patch Links:
> > >> >
> > >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
> > >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
> > >> >
> > >> > —
> > >> > You are receiving this because you are subscribed to this thread.
> > >> > Reply to this email directly, view it on GitHub
> > >> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute the
> > >> thread
> > >> > <
> > >> https://github.com/notifications/unsubscribe-auth/
> > AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia
> > >> >
> > >> > .
> > >> >
> > >>
> > >> —
> > >> You are receiving this because you authored the thread.
> > >> Reply to this email directly, view it on GitHub
> > >> <https://github.com/UglyBlueCat/quickstart-ios/
> > pull/1#issuecomment-356261857>,
> > >> or mute the thread
> > >> <https://github.com/notifications/unsubscribe-auth/ABCFgjHUqu_
> > 1w4KSGNAJ2aNMTUg_-dCqks5tI1F7gaJpZM4RXJia>
> > >> .
> > >>
> > >
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-358496458
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AC4FNGTl3V9obVinkngattqZw22a9fqBks5tLpBjgaJpZM4RXJia
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-364889756>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCFgto-3465Citq5E7Q7QlYHGqU5tYvks5tUBk2gaJpZM4RXJia>
> .
>

UglyBlueCat commented:
Hi Micheal,

I finally got some time to do this for you, but the build was successful
all three times.

Maybe a recent apple update has fixed an issue in macos or xcode.

Robin


Robin Spinks
07710 520048
uglybluecat@googlemail.com

On 12 February 2018 at 18:34, Michael Haney <notifications@github.com>
wrote:

> If you're able to do it, we'd appreciate it and I'd try to use that
> information to solve the problem. For now, after spending 5 full days
> trying to reproduce this, I've moved on to other issues.
>
> On Mon, Feb 12, 2018 at 2:57 AM Robin Spinks <notifications@github.com>
> wrote:
>
> > Hi Mike,
> >
> > Do you still need this?
> >
> > I'm sorry, but my work has become very busy as I approach a deadline.
> >
> > Robin
> >
> >
> > Robin Spinks
> > 07710 520048
> > uglybluecat@googlemail.com
> >
> > On 18 January 2018 at 00:29, Michael Haney <notifications@github.com>
> > wrote:
> >
> > > Hi Robin,
> > >
> > > Odd request: can you please build and run 3 times and upload (or email
> > me)
> > > the output of the 3rd run? I really appreciate your help, and I think
> I'm
> > > getting to the bottom of the issue.
> > >
> > > On Tue, Jan 16, 2018 at 7:29 PM Michael Haney <mikehaney24@gmail.com>
> > > wrote:
> > >
> > > > Thanks for uploading the new log.
> > > >
> > > > You're right--Crashlytics is the way to go, but we definitely don't
> > want
> > > > to there to be any issues for our existing customers of Firebase
> Crash,
> > > so
> > > > I'm trying to fix this.
> > > >
> > > > To that end, I've been trying to reproduce this issue and haven't
> been
> > > > able to. I'll keep trying and ask for more information if I need any.
> > > > Thanks a lot for your help so far!
> > > >
> > > > On Tue, Jan 9, 2018 at 3:46 AM Robin Spinks <
> notifications@github.com>
> > > > wrote:
> > > >
> > > >> Hi Micheal,
> > > >>
> > > >> I've saved the log into
> > > >>
> > > >> https://github.com/UglyBlueCat/quickstart-ios/
> > > blob/master/crashreporting/Build%20CrashReportingExample_
> > > 2018-01-09T11-00-59.rtf
> > > >>
> > > >> I don't know why I didn't see that save button before, but now you
> > have
> > > >> the
> > > >> whole part of the log relevant to the issue.
> > > >>
> > > >> here's the output of the two commands you requested:
> > > >>
> > > >> $ which curl
> > > >> /usr/bin/curl
> > > >> $ curl --version
> > > >> curl 7.54.0 (x86_64-apple-darwin17.0) libcurl/7.54.0 LibreSSL/2.0.20
> > > >> zlib/1.2.11 nghttp2/1.24.0
> > > >> Protocols: dict file ftp ftps gopher http https imap imaps ldap
> ldaps
> > > pop3
> > > >> pop3s rtsp smb smbs smtp smtps telnet tftp
> > > >> Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM
> > NTLM_WB
> > > >> SSL
> > > >> libz HTTP2 UnixSockets HTTPS-proxy
> > > >>
> > > >> I should have told you that we have now installed crashlytics, as
> per
> > > your
> > > >> colleagues suggestion, and it is working fine, so we have removed
> > > firebase
> > > >> crash reporting.
> > > >>
> > > >> We figured google probably wouldn't pursue this issue as a priority,
> > > since
> > > >> they appear to be phasing out firebase crash reporting.
> > > >>
> > > >> You may close this issue if you wish, but I will continue to help
> you
> > > >> resolve it if that is your wish, since there may be others with the
> > > issue
> > > >> who can't (or won't) switch to crashlytics.
> > > >>
> > > >> Thanks,
> > > >> Robin
> > > >>
> > > >>
> > > >> Robin Spinks
> > > >> 07710 520048
> > > >> uglybluecat@googlemail.com
> > > >>
> > > >> On 9 January 2018 at 00:29, Michael Haney <notifications@github.com
> >
> > > >> wrote:
> > > >>
> > > >> > Hello! I'm one of the authors of Firebase Crash Reporting. I'm
> > trying
> > > to
> > > >> > figure out why you're having trouble uploading crashes. Can you
> > accept
> > > >> this
> > > >> > change, re-run, and then update https://github.com/
> > > >> > UglyBlueCat/quickstart-ios/blob/master/crashreporting/
> > > upload_sym_error
> > > >> > with the output of that?
> > > >> >
> > > >> > I was prohibited from updating to High Sierra for a while on my
> work
> > > >> > computer and wasn't able to repro on Sierra. Now that I've finally
> > > been
> > > >> > allowed to upgrade to High Sierra, I tried again and still
> couldn't
> > > >> repro.
> > > >> >
> > > >> > I also just want to say that I appreciate your efforts to give as
> > much
> > > >> > information as possible!
> > > >> >
> > > >> > Lastly, I'm also interested in the output of the following two
> > > commands:
> > > >> > which curl
> > > >> > curl --version
> > > >> >
> > > >> > Thanks!
> > > >> > ------------------------------
> > > >> > You can view, comment on, or merge this pull request online at:
> > > >> >
> > > >> > https://github.com/UglyBlueCat/quickstart-ios/pull/1
> > > >> > Commit Summary
> > > >> >
> > > >> > - Add some diagnostics
> > > >> >
> > > >> > File Changes
> > > >> >
> > > >> > - *M* crashreporting/Firebase/Crash/upload-sym-util.bash
> > > >> > <https://github.com/UglyBlueCat/quickstart-ios/
> pull/1/files#diff-0>
> > > >> > (2)
> > > >> >
> > > >> > Patch Links:
> > > >> >
> > > >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.patch
> > > >> > - https://github.com/UglyBlueCat/quickstart-ios/pull/1.diff
> > > >> >
> > > >> > —
> > > >> > You are receiving this because you are subscribed to this thread.
> > > >> > Reply to this email directly, view it on GitHub
> > > >> > <https://github.com/UglyBlueCat/quickstart-ios/pull/1>, or mute
> the
> > > >> thread
> > > >> > <
> > > >> https://github.com/notifications/unsubscribe-auth/
> > > AC4FNOfig4UFjIFDT9mOWzlVfx_K-mwyks5tIrLXgaJpZM4RXJia
> > > >> >
> > > >> > .
> > > >> >
> > > >>
> > > >> —
> > > >> You are receiving this because you authored the thread.
> > > >> Reply to this email directly, view it on GitHub
> > > >> <https://github.com/UglyBlueCat/quickstart-ios/
> > > pull/1#issuecomment-356261857>,
> > > >> or mute the thread
> > > >> <https://github.com/notifications/unsubscribe-auth/ABCFgjHUqu_
> > > 1w4KSGNAJ2aNMTUg_-dCqks5tI1F7gaJpZM4RXJia>
> > > >> .
> > > >>
> > > >
> > >
> > > —
> > > You are receiving this because you modified the open/close state.
> > > Reply to this email directly, view it on GitHub
> > > <
> > https://github.com/UglyBlueCat/quickstart-ios/
> pull/1#issuecomment-358496458
> > >,
> > > or mute the thread
> > > <
> > https://github.com/notifications/unsubscribe-auth/
> AC4FNGTl3V9obVinkngattqZw22a9fqBks5tLpBjgaJpZM4RXJia
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/UglyBlueCat/quickstart-ios/
> pull/1#issuecomment-364889756>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/ABCFgto-
> 3465Citq5E7Q7QlYHGqU5tYvks5tUBk2gaJpZM4RXJia>
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/UglyBlueCat/quickstart-ios/pull/1#issuecomment-365018163>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC4FNLx4QxnR54juMUAIAMkNWLJw8ykjks5tUIQngaJpZM4RXJia>
> .
>
